### ISSUE TITLE
Terse error messages

### ISSUE THREAD

**Issue by leicray** [openvar/variantValidator#157]
The sequence variant `NM_004006.2:c.93+1G>T` validates without errors or warnings against GRCh38 and GRCh37. However, using version 1 of the reference sequence  produces:

> Error
> Unable to validate the submitted variant `NM_004006.1:c.93+1G>T` against the GRCh38 assembly. The following warnings were returned:
> 
> Please check your submission and re-submit.

`NM_004006.1` was replaced by `NM_004006.2` on 24 May 2009 so it's perhaps understandable that it was not mapped  even onto GRCh37 which was released on 27 Feb 2009. The previous build, hg 18, was released on 3 March 2006 and `NM_004006.1` was presumably mapped to that build.

In summary, the warning seems unnecessarily terse.

**Comment by PeteCausey-Freeman** [comment]
VV now ouputs
```json
        "validation_warnings": [
            "Required information for NM_004006.1 is missing from the Universal Transcript Archive",
            "Query gene2transcripts with search term NM_004006 for available transcripts"
```
Will check the interactive tool functionality

**Comment by PeteCausey-Freeman** [comment]
Error
Unable to validate the submitted variant NM_004006.1:c.93+1G>T against the GRCh38 assembly. The following warnings were returned:
This is an unhandled error by the interactive tool for example

**Comment by PeteCausey-Freeman** [comment]
https://github.com/openvar/VVweb/commit/59e577653ba95c93d9ba31d94800eebf85143767
closes
